sgx-types/src/lib.rs

```diff
@@ -84,21 +85,31 @@ macro_rules! testaso {
     };
 }
 
+/// Makes SGX ATTRIBUTES type available (38.7.1).
 pub mod attr;
+/// Makes SGX MISCSELECT type available (38.7.2).
 pub mod misc;
+/// Makes SGX SECINFO (describing properties of an enclave page) available (38.11).
 pub mod page;
+/// Makes SGX SECS (Enclave Control Structure) type available (38.7).
 pub mod secs;
+/// Makes SGX SIGSTRUCT type availalbe (38.13).
 pub mod sig;
+/// Makes SGX State Save Area type available (38.9).
 pub mod ssa;
+/// Makes SGX TCS (Thread Control Struct) type availalbe (38.8).
 pub mod tcs;
 
 use core::fmt::Debug;
 use core::ops::BitAnd;
 
+/// Succinctly describes a masked type, ex. masked Attributes or masked MiscSelect.
```

Comment:
Is `ex.` equivalent to `e.g.`?

Reply:
Also I don't understand how `MiscSelect` relates to this struct

Reply:
> Is `ex.` equivalent to `e.g.`?

Yes. : ) I guess other people use `e.g.` in our repos, so I've changed it for consistency.